biogeochem/FatesLitterMod.F90

```diff
@@ -424,6 +424,73 @@ function GetTotalLitterMass(this) result(total_mass)
     
     return
   end function GetTotalLitterMass
-
+  
+  ! =====================================================
+
+  subroutine adjust_SF_CWD_frac(dbh,ncwd,SF_val_CWD_frac,SF_val_CWD_frac_adj)
+     
+     !DESCRIPTION
+     !Adjust  the partitioning of struct + sawp into cwd pools based on 
+     !cohort dbh. This avoids struct and sapw from small cohorts going to
+     !fuel classes that are larger than their dbh. Here, struct + sapw are sent to the cwd
+     !class consistent with fuel class diameter thresholds (Fosberg et al., 1971;
+     !Rothermel, 1983)
+
+     !ARGUMENTS
+     real(r8), intent(in)               :: dbh !dbh of cohort [cm]
+     type(integer), intent(in)          :: ncwd !number of cwd pools
+     real(r8), intent(in)               :: SF_val_CWD_frac(:) !fates parameter specifying the
+                                                              !fraction of struct + sapw going
+                                                              !to each CWD class
+     real(r8), intent(out)              :: SF_val_CWD_frac_adj(:) !Updated cwd paritions
+     !
+     !LOCAL VARIABLES
+     !These diameter ranges are based on work by Fosberg et al., 1971 
+     
+     real(r8), parameter :: lb_max_diam        = 7.6 !max diameter [cm] for large branch
+     real(r8), parameter :: sb_max_diam        = 2.5 !max diameter [cm] for small branch
+     real(r8), parameter :: twig_max_diam      = 0.6 !max diameter [cm] for twig
+     !------------------------------------------------------------------------------------
+
+     
+     SF_val_CWD_frac_adj = SF_val_CWD_frac
+     
+     !If dbh is larger than max size of a large branch (1,000 hr fuel) then we don't change
```

Comment:
If DBH is larger than the max size of a large branch (and is thus a 1000hr fuel)? Wording here is confusing. 7.6 is the max boundary for 100 hour fuels in Rothermel.  Thank you